opensrp-chw-core/src/test/java/org/smartregister/chw/core/activity/impl/CoreFamilyPlanningMemberProfileActivityImpl.java

```diff
@@ -0,0 +1,40 @@
+package org.smartregister.chw.core.activity.impl;
```

Comment:
Could you explain why this class is needed?

Reply:
Hi. This an implementation of an Abstract class for tests purposes